### DISCUSSION TITLE
[Feature Request] Add visual indication of driver cam audio

### DISCUSSION THREAD

**Discussion by n4ppy** [f1multiviewer/issue-tracker#565] [Feature requests]
Sometimes driver cam audio can get a bit hectic when following multiple drivers. Two things that could help:
1: A (speaker) icon somewhere in the video feed or a border around the driver name/number when the car telemetry is set on.
-This wil give a visual clue what radio is on. This can help decode who is talking
-Challenge is probably to find the right trigger against the background noise.

~~2: An option to pan the audio between left and right so you can put driver 1 left and driver 2 right~~
~~-Will help pull the audio streams apart when multiple drivers/team are talking.~~

**Comment by jeppeolesen** [comment]
Redirecting the audio pan part of the request to #184
